packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -116,6 +120,39 @@ const MarketPlacePage = () => {
     }
   }, [toggleNotification, isInDevelopmentMode]);
 
+  if (!isOnline) {
+    return (
+      <Layout>
+        <PageHeader />
+        <Flex
+          width="100%"
+          direction="column"
+          alignItems="center"
+          justifyContent="center"
+          paddingTop={11}
+        >
+          <Box paddingTop={11} paddingBottom={1}>
+            <Typography textColor="neutral700" variant="alpha">
+              {formatMessage({
+                id: 'admin.pages.MarketPlacePage.offline.title',
+                defaultMessage: 'You are offline',
+              })}
+            </Typography>
+          </Box>
+          <Box paddingBottom={3}>
+            <Typography textColor="neutral700" variant="epsilon">
+              {formatMessage({
+                id: 'admin.pages.MarketPlacePage.offline.subtitle',
+                defaultMessage: 'You need to be connected to the Internet to access Strapi Market.',
+              })}
+            </Typography>
+          </Box>
+          <Box as="img" src={offlineCloud} alt="offline" width={11} height={11} />
```

Comment:
Looks like this would be 64x64 instead of 88x88

Reply:
Yeah you are right, but this is as big as the design system allows isn't it? If I understand correctly this is what he have available: `['0px', '4px', '8px', '12px', '16px', '20px', '24px', '32px', '40px', '48px', '56px', '64px']`

Reply:
Yes but I just wanted to point it out in case Raff still wants the bigger size, because then we could also use custom CSS

Reply:
I guess for this and the padding on the container for 120px I can just use custom css...but I'm always a bit confused when it is acceptable to do that.

Reply:
ok sounds good I asked Raff and will update according to what he thinks is best

Reply:
I don't know either when it's okay 🤷